src/Microsoft.IdentityModel.Protocols/Configuration/HttpDocumentRetriever.cs

```diff
@@ -81,7 +81,7 @@ public HttpDocumentRetriever(HttpClient httpClient)
                 throw LogHelper.LogArgumentNullException("address");
 
             if (!Utility.IsHttps(address) && RequireHttps)
-                throw LogHelper.LogExceptionMessage(new ArgumentException(LogMessages.IDX10108, address));
+                throw LogHelper.LogExceptionMessage(new ArgumentException(LogMessages.IDX10108, nameof(address)));
```

Comment:
The idea here was that IDX10108 should have a parameter that has the 'address' value. Include the value for 'address' in the error string.